src/entities/protocols/seaport.ts

```diff
@@ -14,6 +17,9 @@ export type SeaportData = {
   items: Order[]
   recipient: string // address
   version: SeaportVersion
+  inputTokenApproval?: string
+  inputTokenPermit?: Permit2Permit
+  inputTokenTransfer?: Permit2TransferFrom
```

Comment:
I'm wondering if these merit code comments.... and how they work together.
Is there a situation where `inputTokenApproval` would be false, but the other two fields are filled in?

Reply:
We're leaving approvals hanging, but yeah I agree it would be good to have a detailed comment here specifically about the `inputTokenApproval` as well as it's not intuitive that it stands for the approval of the opensea conduit by the Universal router for the ERC20. Someone else might think that it's for approving their tokens for universal router